### PR TITLE
Move docker-compose.yml to LiskHQ/lisk, remove confd template, use node:8 - Closes #2237, #2305, #2306

### LINKED ISSUE #2237: Move docker-compose.yml file to lisk repository
### Expected behavior

Once https://github.com/LiskHQ/lisk/pull/2234 and https://github.com/LiskHQ/lisk-docker/pull/100 are merged we will have one `Dockerfile` and one `docker-compose.yml` as well as the ability to run any network configuration with the `-n` switch. Under those conditions it makes sense to move the `docker-compose.yml` file from the LiskHQ/lisk-docker repository to LiskHQ/lisk.

### Actual behavior

For historical reasons `docker-compose.yml` is hosted at LiskHQ/lisk-docker just like the `Dockerfile` used to be.

### Steps to reproduce

Look at both LiskHQ/lisk and LiskHQ/lisk-docker repositories.

### Which version(s) does this affect? (Environment, OS, etc...)

Docker using lisk core >= 1.1

### PR DESCRIPTION
**Pull request by ghost**
### What was the problem?

`docker-compose.yml` is in LiskHQ/lisk repo, just like the `Dockerfile`

### How did I fix it?

Moved `docker-compose.yml` from LiskHQ/lisk-docker to LiskHQ/lisk

### How to test it?

```
cd docker/
cp .env.development .env
make [coldstart]
```

### Review checklist

* The PR solves #2237 
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated

### CODE REVIEW

**Review by diego-G — DISMISSED** [review]
I thought the purpose of this PR was to remove the templates from https://github.com/LiskHQ/lisk/tree/2237-docker-compose-yml/docker_files/etc/confd . Have you changed your mind @fchavant ?

**Comment by fchavant** [comment]
@diego-G I forgot to stage the deletions, sorry.

**Comment by fchavant** [comment]
@diego-G this will happen right after this PR has part of https://github.com/LiskHQ/lisk/issues/2305

**Comment by diego-G** [comment]
@fchavant could you tackle both issues in the same PR or is there any obstacle?

**Comment by fchavant** [comment]
@diego-G can do.

**Comment by diego-G** [comment]
Please, update the title adding both issues.
Another question is, how will the DB containers react when we update to `1.2.0` images that has psql 10? The DB will be automatically migrated or the nodes are gonna rebuild the blockchain data?

File: docker/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Using docker-compose
+
```
Comment (ManuGowda) [inline docker/README.md]: are you missing the example commands here?

**Review by ManuGowda — APPROVED** [review]
Made a comment, otherwise the PR looks good to me.

File: docker/README.md (hunk 1 above)
Comment (fchavant) [reply docker/README.md]: No, there are some below after the prerequisite of creating a `.env` from one of the sample files; do you wish to see more examples?

**Comment by MaciejBaj** [comment]
Please open the corresponding issue on lisk-docker - after releasing 1.2.0 `lisk-docker` repo can be removed.

**Comment by fchavant** [comment]
@MaciejBaj https://github.com/LiskHQ/lisk-docker/issues/103

**Review by MaciejBaj — APPROVED** [review]

**Pull request merged**